tools/typescript2fable.js

```diff
@@ -310,9 +335,12 @@ function printModule(prefix) {
     return function(mod) {
         var template = prefix + templates.module
             .replace("[NAME]", escapeKeyword(mod.name));
-            
-        template = append(template, mod.interfaces.map(
-            printInterface(prefix + "    ", mod.name)).join("\n\n"));
+
+        var interfaces = mod.interfaces && mod.interfaces.length > 0
+                            ? mod.interfaces.map(printInterface(prefix + "    ", mod.name)).join("\n\n")
+                            : "";
```

Comment:
What are the edge cases this is trying to solve? `mod.interfaces` should never be `undefined` and executing `map` + `join` on an empty array should give an empty string so the check `mod.interfaces.length > 0` shouldn't be necessary